src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splitSourceString = sourceString
+    .split(';')
+    .map(a => a.split(':'));
```

Comment:
Create meaningful names
```suggestion
    .map(keyValuePair => keyValuePair.split(':'));
```

Reply:
i am still very bad at naming thing but feel like improved a bit